lib/kafka_base_producer.js

```diff
@@ -297,13 +306,14 @@ KafkaBaseProducer.prototype._produce = function _produce(msg, callback) {
 
     function onResponse(err, result) {
         if (callback) {
-            callback(err, result);
+            return callback(err, result);
```

Comment:
since there is statement after this if() branch. the 'return' looks not quite safe.

Reply:
The lint rule is very strict on enforcing "return after callback"
For this case I will disable the rule here and make sure behavior is the same.